config/src/config/network_config.rs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::{
+    chain_id::ChainId,
```

Comment:
So do we duplicate this across every network config or insert it into the BaseConfig?

Reply:
The BaseConfig is probably a better place to put it, either way it has to be trickled down to handshake.  I was considering putting it into the NetworkContext, but realize it probably doesn't have to be logged all that much.